scripts/oak.fsx

```diff
@@ -0,0 +1,280 @@
+#r "nuget: Thoth.Json.System.Text.Json"
```

Comment:
Useful to test out the local compiler and view changes that Fantomas Tools might not yet have.